cmd/drone-server/drone.go

```diff
@@ -236,6 +244,14 @@ func main() {
 		auth.POST("", server.GetLogin)
 	}
 
+	redirects := r.Group("/redirect")
+	{
+		redirects.Use(server.SetDatastore(store))
+		redirects.Use(server.SetRepo())
+
+		redirects.GET("/:owner/:name/commit/:sha", server.RedirectSha)
```

Comment:
I'd like to move this function up w/ the repo group instead if you don't mind. And also use `/s/commit/commits/g`

```diff
repo.GET("/pulls/:number", server.GetPullRequest)
+repo.GET("/commits/:sha", server.GetCommit)
```